WgServerforWindows/Models/ClientConfiguration.cs

```diff
@@ -57,16 +57,16 @@ public ClientConfiguration(ClientConfigurationList parentList)
                         .Select(a => IPNetwork.Parse(a.Trim()))
                         .ToList(); // Prevent multiple enumeration
 
-                    var currentClientAddresses = prop.Value
+                    var currentClientAddresses = (prop.Value?

```

Comment:
While `(prop.Value ?? "")` in case of `prop.Value == null` causes an empty list at the end of the chain, I'm ok with this version if you'd prefer.

Reply:
I think we're splitting hairs here, but mine might be slightly more efficient. We won't allocate a stack for each of those LINQ methods, and we won't instantiate a new `IEnumerable` since `Empty` is static. Any other thoughts on this point?

Reply:
Nope. I didn't know that. So we can simply ignore this, and thanks for letting me know!